packages/desktop-client/src/components/autocomplete/Autocomplete.tsx

```diff
@@ -164,7 +164,7 @@ type SingleAutocompleteProps = {
   openOnFocus?: boolean;
   getHighlightedIndex?: (suggestions) => number | null;
   highlightFirst?: boolean;
-  onUpdate: (id: unknown, value: string) => void;
+  onUpdate?: (id: unknown, value: string) => void;
```

Comment:
This is optional - it's hardly ever specified - found it when looking in another area.